### DISCUSSION TITLE
挂载google drive 报错

### DISCUSSION THREAD

**Discussion by jokenee** [AlistGo/alist#502] [Q&A]
ERRO[2022-02-07 12:38:59] init account [Public] error:[Post "https://www.googleapis.com/oauth2/v4/token": dial tcp: i/o timeout] 

挂载不上啥问题呢

**Comment by xhofe** [comment]
能访问Google？

**Reply by jokenee** [reply]
谢谢回复
我  docker部署的

**Reply by xhofe** [reply]
报错信息就是访问不了Google。

**Reply by orrzz** [reply]
遇到同样问题，是否已解决？

**Reply by 95121808** [reply]
一样超时，不懂怎么处理中。。。

**Reply by ArthurZhou** [reply]
2023.2.9 问题依然存在
`2023/02/09 14:51:22.157208 ERROR RESTY Post "https://www.googleapis.com/oauth2/v4/token": context deadline exceeded (Client.Timeout exceeded while awaiting headers), Attempt 4`

**Reply by anwen-anyi** [reply]
> > 2023.2.9 问题依然存在 `2023/02/09 14:51:22.157208 ERROR RESTY Post "https://www.googleapis.com/oauth2/v4/token": context deadline exceeded (Client.Timeout exceeded while awaiting headers), Attempt 4`
> 
> 都提示你了 你自己的网络问题.......
文档里面的提示也说的很明白~

**Reply by ArthurZhou** [reply]
> > > 2023.2.9 问题依然存在 `2023/02/09 14:51:22.157208 ERROR RESTY Post "https://www.googleapis.com/oauth2/v4/token": context deadline exceeded (Client.Timeout exceeded while awaiting headers), Attempt 4`
> > 
> > 
> > 都提示你了 你自己的网络问题.......
> > 文档里面的提示也说的很明白~

所以没有办法在国内的服务器上访问google drive是吗

**Reply by anwen-anyi** [reply]
> > > > 2023.2.9 问题依然存在 `2023/02/09 14:51:22.157208 ERROR RESTY Post "https://www.googleapis.com/oauth2/v4/token": context deadline exceeded (Client.Timeout exceeded while awaiting headers), Attempt 4`
> > > 
> > > 
> > > 都提示你了 你自己的网络问题.......
> > > 文档里面的提示也说的很明白~
> 
> 所以没有办法在国内的服务器上访问google drive是吗

自己想办法让Alist 吃到代理也不是不能，我就是这样做的。
